chainerx_cc/chainerx/routines/connection.cc

```diff
@@ -362,8 +362,12 @@ Array Linear(const Array& x, const Array& w, const absl::optional<Array>& b, uin
         }
     }
 
-    BackwardBuilder bb = has_bias ? BackwardBuilder{"linear", {x_matrix, w, b_matrix}, out_matrix}
-                                  : BackwardBuilder{"linear_nobias", {x_matrix, w}, out_matrix};
+    // NOTE: 1. BackwardBuilder's move assignment is deleted, use it will cause compilation error with MSVC.
```

Comment:
I asked about move assignment operator because of this comment.

So, you mean defining the move assignment operator does not change error message at all, right?
(comparing with the original report https://github.com/chainer/chainer/issues/8192#issue-499526799).

Reply:
yeah

Reply:
>BackwardBuilder's move assignment is deleted,

Hm, then this clause sounds irrelevant.